Resources/Locale/en-US/criminal-records/general-criminal-records.ftl

```diff
@@ -0,0 +1,46 @@
+general-criminal-record-console-window-title = Criminal Records Computer
+general-criminal-record-console-select-record-info = Select a record on the left.
+general-criminal-record-console-empty-state = No records found!
+general-criminal-record-console-no-record-found = No record was found for the selected person.
+
+general-criminal-record-arrest-button = Arrest
+general-criminal-record-release-button = Release
+
+general-criminal-record-reason-placeholder = Reason
+
+general-criminal-record-permission-denied = Permission denied
+
+general-criminal-record-console-record-age = Age: {$age}
+general-criminal-record-console-record-title = Job: {$job}
+general-criminal-record-console-record-species = Species: {$species}
+general-criminal-record-console-record-gender = Gender: {$gender}
+general-criminal-record-console-record-fingerprint = Fingerprint: {$fingerprint}
+general-criminal-record-console-record-dna = DNA: {$dna}
+general-criminal-record-console-record-status = Status: {$status}
+
+## Security channel notifications
+### On Arrest/Release button pressed
+
+general-criminal-record-console-detained-with-reason = {$name} has been detained for {$reason} by {$goodguyname}
+general-criminal-record-console-detained-without-reason = {$name} has been detained by {$goodguyname}
+
+general-criminal-record-console-undetained-with-reason = {$name} has been released from the detention for {$reason} by {$goodguyname}
+general-criminal-record-console-undetained-without-reason = {$name} has been released from the detention by {$goodguyname}
+
+## On status changed
+
+general-criminal-record-console-wanted-with-reason = {$name} is wanted for {$reason} by {$goodguyname}
+general-criminal-record-console-wanted-without-reason = {$name} is wanted by {$goodguyname}
+
+general-criminal-record-console-not-wanted-with-reason = {$name} is not wanted anymore for {$reason} by {$goodguyname}
+general-criminal-record-console-not-wanted-without-reason = {$name} is not wanted anymore by {$goodguyname}
+
+
+## Filters
+
+general-criminal-record-for-filter-line-placeholder = Input text and press "Enter"
```

Comment:
Perhaps `Input name, DNA / Fingerprint prefix and press "Enter"`

Reply:
1. It wasn't made by me
2. I think it's too long for a placeholder